index.js

```diff
@@ -307,15 +308,35 @@ var UserAttributeHandler = require('../../../src/user-attribute-handler');
     }
 
     function register(config) {
-        if (config.kits) {
+        if (!config) {
+            window.console.log('You must pass a config object to register the kit ' + name);
```

Comment:
Curious to know why we're specifying `window.console.log` rather than `console.log`. I know they're the same thing, but I was just wondering why you're doing it the "long way".